types/react-input-mask/index.d.ts

```diff
@@ -58,6 +58,8 @@ export interface Props extends React.InputHTMLAttributes<HTMLInputElement> {
    * * nextState: Input state with applied mask. Contains value and selection fields.
    */
   beforeMaskedStateChange?(states: BeforeMaskedStateChangeStates): InputState;
+
+  children?: () => JSX.Element;
```

Comment:
Thanks for doing this, @jsfelix!  I also need this change since upgrading to react 18 and don't want to upgrade to the alpha version of react-input-mask.

You might want to add the following params:
```suggestion
  children?: (inputProps: React.InputHTMLAttributes<HTMLInputElement>) => JSX.Element;
```

See [v2 docs](https://github.com/sanniassin/react-input-mask/tree/v2#children--function), [source](https://github.com/sanniassin/react-input-mask/blob/e1b3a7ca1b8728e518a3f1dc3e9743916d73a272/src/index.js#L570)

Reply:
you're right. Thanks!